File: src/main/java/com/jafupy/CreateStorageDimension.java
package com.jafupy;

import net.fabricmc.api.ModInitializer;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class CreateStorageDimension implements ModInitializer {
	public static final String ModID = "create-storage-dimension";
    public static final Logger LOGGER = LoggerFactory.getLogger(ModID);

	@Override
	public void onInitialize() {

		LOGGER.info("Hello Fabric world!");
	}
}

File: src/main/java/com/jafupy/CreateStorageDimension.kt
package com.jafupy

import net.fabricmc.api.ModInitializer
import org.slf4j.Logger
import org.slf4j.LoggerFactory

class CreateStorageDimension : ModInitializer {
    override fun onInitialize() {
        LOGGER.info("Hello Fabric world!")
    }

    companion object {
        const val ModID: String = "create-storage-dimension"
        val LOGGER: Logger = LoggerFactory.getLogger(ModID)
    }
}